ecc/internal/gpoint/base.go

```diff
@@ -50,6 +50,101 @@ type {{.Name}}Affine struct {
 	X, Y {{.CType}}
 }
 
+// {{.Name}}ZZZ parameterized jacobian coordinates (x=X/ZZ, y=Y/ZZZ, ZZ**3=ZZZ**2)
+type {{.Name}}ZZZ struct {
```

Comment:
cf comment on generated file: do we want to export that (if yes find better name) if not, lowercase letter